frontend/src/components/authentication/LoginForm.jsx

```diff
@@ -21,33 +24,58 @@ const styles = {
     display: "block",
     margin: "24px auto"
   },
+  loginError: {
+    color: "red"
```

Comment:
Do we have a specific hash value @joeyhua

Reply:
I think that we can update that during the design session with @joeyhua.
What do you think?